libs/data-contracts/signatureSchemas.js

```diff
@@ -215,7 +215,7 @@ schemas.addIPLDBlacklist = [
 ]
```

Comment:
can you overwrite this file with the latest signatureSchemas file from data-contracts to get the above changes?

Reply:
yes